index.html

```diff
@@ -3514,6 +3514,58 @@ <h3>Reserved Extension Points</h3>
         </p>
 
       </section>
+
+      <section class="normative">
+        <h3>Ecosystem Compatibility</h3>
+
+        <p>
+There are a number of digital credential formats that do not natively use the
+data model provided in this document, but are aligned with a number of concepts
+in this specification. At the time of publication, examples of these digital
+credential formats include pure JSON Web Tokens (JWTs), CBOR Web Tokens (CWTs),
+ISO-18013-5 (mDLs), and Authentic Chained Data Containers (ACDCs).
+        </p>
+
+        <p>
+If conceptually aligned digital credential formats can be transformed to the
+data model according to the rules provided in this section, they are considered
+<em>"compatible with the Verifiable Credentials ecosystem"</em>, and the result
+of the transformation, which is a <a>conforming document</a>, is a
+<a>verifiable credential</a>. Specifications that contain transformations to the
+data model described by this specification:
+        </p>
+
+        <ul>
+          <li>
+MUST identify if the transformation to this data model is uni-directional or
+bi-directional.
+          </li>
+          <li>
+MUST preserve the `@context` values when performing bi-directional
+transformation.
```

Comment:
agree with this statement, but wondering if we can be a bit more helpful for implementers in applying guidance on _how_ to achieve this requirement. would a link to an implementers guide section, or note about `@vocab` doing this automagically be helpful?

Reply:
`@vocab` is not enough to preserve the v2 context, because it has several processign directives that are applied to specific predicates, that alter the shape of the RDF graph... for example `@id`, `@type`, `@container`, etc...